src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/ValuesColumn.js

```diff
@@ -18,6 +18,16 @@ export const ValuesColumn = ({ action, actionIndex, onChange, option }) => {
       value={action.value}
       onChange={e => onChange({ actionIndex, value: e.target.value, fieldName: FIELD_VALUE_KEY })}
       data-testid={`input-email-${actionIndex}`}
+      aria-label="Text field"
+    />
+  );
+
+  const renderTextArea = () => action.type === CONTROL_TYPES.TEXTAREA && (
+    <TextArea
+      value={action.value}
+      onChange={e => onChange({ actionIndex, value: e.target.value, fieldName: FIELD_VALUE_KEY })}
+      data-testid={`input-textarea-${actionIndex}`}
+      aria-label="Text area"
```

Comment:
I think the `aria-label` should be translated here and elsewhere in the PR.

Reply:
Hi @Dmytro-Melnyshyn, I've checked our ui modules and there are some modules with translations and some without. But I agree that it makes sense, so I've fixed. Thanks!